releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,12 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v7.0.0 - November, 20 2023
+**Breaking changes**
+  - Updated to fail the task if any Key Vault references cannot be resolved or if the resolved values are invalid to be set as the pipeline secret.
```

Comment:
Ideally when we call out a breaking change, we inform how to work around it. In this case the user needs to enable the option to treat key vault errors as warnings.

Reply:
That's a good point. I have also added more information on a workaround, let me know if thats okay